pkg/plugin/cmd.go

```diff
@@ -196,13 +228,37 @@ func (o *DebugOptions) Validate() error {
 	return nil
 }
 
+// Run run
 func (o *DebugOptions) Run() error {
-
 	pod, err := o.CoreClient.Pods(o.Namespace).Get(o.PodName, v1.GetOptions{})
 	if err != nil {
 		return err
 	}
+	agents, err := o.CoreClient.Pods(v1.NamespaceAll).List(v1.ListOptions{
+		LabelSelector: "app=debug-agent",
```

Comment:
debug-agent is not a strictly conventional name, prefer reading from configurations here

Reply:
done